File: Python Dasar/Contoh Fungsi/Contoh Fungsi (Function)/Fungsi (Function) Hitung.py
print("Menu: \n")
print("1. Pertambahan ")
print("2. Pengurangan ")
print("3. Perkalian   ")
print("4. Pembagian \n")
pil = input("Masukkan Pilihan : ")

def pertambahan(a,b):
   print("--------- +")
   print("Hasil : ",a+b)
def pengurangan(a,b):
   print("--------- -")
   print("Hasil : ",a-b)
def perkalian(a,b):
   print("--------- *")
   print("Hasil : ",a*b)
def pembagian(a,b):
   print("--------- /")
   print("Hasil : ",a/b)

if pil == "1":
   print("\nSilahkan Masukkan Angka \n")
   a = int(input("Angka : "))
   b = int(input("Angka : "))
   pertambahan(a,b)
elif pil == "2":
   print("\nSilahkan Masukkan Angka \n")
   a = int(input("Angka : "))
   b = int(input("Angka : "))
   pengurangan(a,b)
elif pil == "3":
   print("\nSilahkan Masukkan Angka \n")
   a = int(input("Angka : "))
   b = int(input("Angka : "))
   perkalian(a,b)
elif pil == "4":
   print("\nSilahkan Masukkan Angka \n")
   a = int(input("Angka : "))
   b = int(input("Angka : "))
   pembagian(a,b)
else:
   print("Masukkan pilihan (1..4)")

File: C++ Dasar/Contoh Fungsi/Fungsi (Function) Hitung.cpp
#include <iostream>
using namespace std;

void pertambahan(int a, int b) {
	cout << "--------- +" << endl;
	cout << "Hasil : " << a + b << endl;
}

void pengurangan(int a, int b) {
	cout << "--------- -" << endl;
	cout << "Hasil : " << a - b << endl;
}

void perkalian(int a, int b) {
	cout << "--------- *" << endl;
	cout << "Hasil : " << a * b << endl;
}

void pembagian(int a, int b) {
	cout << "--------- :" << endl;
	cout << "Hasil : " << a / b << endl;
}

int main() {
	int a, b;
	int pilihan;

	cout << "Menu\n" << endl;
	cout << "1. Pertambahan " << endl;
	cout << "2. Pengurangan " << endl;
	cout << "3. Perkalian" << endl;
	cout << "4. Pembagian \n" << endl;
	cout << "Masukkan Pilihan : ";
	cin >> pilihan;


	if (pilihan == 1) {
		cout << "\nSilahkan Masukkan Angka \n" << endl;
		cout << "Angka : ";
		cin >> a;
		cout << "Angka : ";
		cin >> b;
		pertambahan(a, b);
	}
	else if (pilihan == 2) {
		cout << "\nSilahkan Masukkan Angka \n" << endl;
		cout << "Angka : ";
		cin >> a;
		cout << "Angka : ";
		cin >> b;
		pengurangan(a, b);
	}
	else if (pilihan == 3) {
		cout << "\nSilahkan Masukkan Angka \n" << endl;
		cout << "Angka : ";
		cin >> a;
		cout << "Angka : ";
		cin >> b;
		perkalian(a, b);
	}
	else if (pilihan == 4) {
		cout << "\nSilahkan Masukkan Angka \n" << endl;
		cout << "Angka : ";
		cin >> a;
		cout << "Angka : ";
		cin >> b;
		pembagian(a, b);
	}
	else {
		cout << "Masukkan pilihan (1..4)" << endl;
	}

	return 0;

}
